app/assets/stylesheets/common/base/group.scss

```diff
@@ -107,12 +107,21 @@ table.group-members {
   width: 100%;
   table-layout: fixed;
 
-  th:first-child {
+  .bulk-select {
+    padding: 10px;
```

Comment:
I don't think you need to specify padding for just this column, it already inherits padding from `td`.